woodwork/statistics_utils.py

```diff
@@ -306,3 +338,79 @@ def _get_value_counts(dataframe, ascending=False, top_n=10, dropna=False):
         values = list(df.to_dict(orient="index").values())
         val_counts[col] = values
     return val_counts
+
+
+def _get_top_values_numeric(series, _range):
+    """Get the most frequent values in a range of integers. Will ignore
+        null values.
```

Comment:
This function isn't really getting the top numeric values. It's probably more accurate to say something like "Get the frequencies of the values in a range of integers" since this is just grabbing all the frequencies and not sorting them in any way currently.

For example the column `'age': pd.Series([33, 34, 36, 34], dtype='Int64'),` produces this dictionary:

```python
'top_values': [{'count': 1, 'value': 33},
   {'count': 2, 'value': 34},
   {'count': 0, 'value': 35},
   {'count': 1, 'value': 36}]},
```

Reply:
Updated this function name and docstring - output is actually sorted by the values defined in the input range.